tensorflow/core/kernels/BUILD

```diff
@@ -1608,7 +1657,9 @@ tf_kernel_libraries(
         "//tensorflow/core:nn_grad",
         "//tensorflow/core:nn_ops_op_lib",
         "//third_party/eigen3",
-    ],
+    ] + if_not_windows([
```

Comment:
I'm confused why this is treated differently from the `tf_kernel_libraries()` target, and thus not in `:math_not_windows`. (It may have been in the wrong place originally, but it'd be good to reduce the complexity if possible.)

I believe it should be possible to remove `:depthwise_conv_op` from the deps here, and move it into the `prefixes` of `:math_not_windows`. Does that make sense?